realm/realm-library/src/androidTest/kotlin/io/realm/RealmModelManagedSetTester.kt

```diff
@@ -421,6 +425,38 @@ class RealmModelManagedSetTester<T : Any>(
 
     override fun hasListeners() = Unit
 
+    override fun aggregations() {
+        if (mixedType == null){
```

Comment:
Maybe just a consequence of the complexity of the tester infrastructure, but it seems a bit hack-ish to select between the two cases like this. At least swap the if so that it is easy discoverable what the reasoning is. 
```
if (mixedType != null){
       // Aggregations on Mixed type are not supported
       managedTester.aggregations()
} else {
       ... 
```